wbooks/src/app/components/BookList/styles.tsx

```diff
@@ -0,0 +1,13 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    padding: 15
```

Comment:
I would like you to add more books until you need to scroll this list and see tell me happens and how could you solve that 😄